dotnet/src/SemanticKernel.Abstractions/SkillDefinition/ParameterView.cs

```diff
@@ -37,6 +37,11 @@ public string Name
     /// </summary>
     public string? DefaultValue { get; set; }
 
+    /// <summary>
+    /// Parameter type.
+    /// </summary>
+    public string? Type { get; set; }
```

Comment:
I assume we expect this type will be limited to a few options. Can we use the fake enum trick here (define a type that looks like an enum but returns strings, and is extensible in case we miss one)?
Examples: 
 - https://github.com/microsoft/semantic-kernel/blob/main/dotnet/src/SemanticKernel.Abstractions/AI/ChatCompletion/AuthorRole.cs
 - https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpmethod?view=net-7.0
 - https://github.com/dotnet/runtime/blob/main/src/libraries/System.Net.Http/src/System/Net/Http/HttpMethod.cs

Reply:
@shawncal please have a look at the implementation.